api/src/core/adapters/dbApi/kysely/kysely.database.ts

```diff
@@ -93,6 +95,7 @@ type SoftwareExternalDatasTable = {
     keywords: JSONColumnType<string[]> | null;
     programmingLanguages: JSONColumnType<string[]> | null;
     applicationCategories: JSONColumnType<string[]> | null;
+    referencePublication: JSONColumnType<SILL.ScholarlyArticle[]> | null;
```

Comment:
si on est sur un tableau, essayons de garder le pluriel, comme les autres 
```suggestion
    referencePublications: JSONColumnType<SILL.ScholarlyArticle[]> | null;
```